packages/flutter/lib/src/rendering/object.dart

```diff
@@ -3714,6 +3714,11 @@ abstract class RenderObject with DiagnosticableTreeMixin implements HitTestTarge
   /// object, for accessibility purposes.
   Rect get semanticBounds;
 
+  /// Wether the semantic bounding box will be clipped by [parent].
+  ///
+  /// Defaults to true,
```

Comment:
```suggestion
  /// Defaults to true.
```

Reply:
Ohoh, good catch! I probably need glasses soon because I had to look closely to see the difference. :smile: